### ISSUE TITLE
Error PHP : Array to string conversion at ... #569

### ISSUE THREAD

**Issue by zouma83** [nextcloud/social#1173]
**Describe the bug**
publication error
spam publication ? from ?
as result many entries in logs :

[PHP] Error: Error: Array to string conversion at /var/www/nextcloud/apps/social/lib/Model/ActivityPub/ACore.php#569 at <<closure>>

 0. <<closure>>
    OC\Log\ErrorHandler::onError(8, "Array to string conversion", "/var/www/nextcl ... p", 569, {as: 8,value: "< ... e})
 1. /var/www/nextcloud/apps/social/lib/Model/ActivityPub/ACore.php line 569
    strip_tags("<p><span class= ... >", ["a","p","span","br"])
 2. /var/www/nextcloud/apps/social/lib/Model/ActivityPub/ACore.php line 498
    OCA\Social\Model\ActivityPub\ACore->validateEntryString(8, "<p><span class= ... >")
 3. /var/www/nextcloud/apps/social/lib/Model/ActivityPub/Stream.php line 428
    OCA\Social\Model\ActivityPub\ACore->validate(8, "content", {id: "https://so ... "}, "")
 4. /var/www/nextcloud/apps/social/lib/Model/ActivityPub/Object/Note.php line 185
    OCA\Social\Model\ActivityPub\Stream->importFromDatabase({id: "https://so ... "})
 5. /var/www/nextcloud/apps/social/lib/Db/StreamRequestBuilder.php line 211
    OCA\Social\Model\ActivityPub\Object\Note->importFromDatabase({id: "https://so ... "})
 6. /var/www/nextcloud/apps/social/vendor/daita/my-small-php-tools/lib/Db/ExtendedQueryBuilder.php line 538
    OCA\Social\Db\StreamRequestBuilder->parseStreamSelectSql({id: "https://so ... "}, OCA\Social\Db\SocialQueryBuilder {})
 7. /var/www/nextcloud/apps/social/lib/Db/StreamRequestBuilder.php line 192
    daita\MySmallPhpTools\Db\ExtendedQueryBuilder->getRows([OCA\Social\Db\S ... "])
 8. /var/www/nextcloud/apps/social/lib/Db/StreamRequest.php line 665
    OCA\Social\Db\StreamRequestBuilder->getStreamsFromRequest(OCA\Social\Db\SocialQueryBuilder {})
 9. /var/www/nextcloud/apps/social/lib/Service/HashtagService.php line 173
    OCA\Social\Db\StreamRequest->getNoteSince(1608094202)
10. /var/www/nextcloud/apps/social/lib/Service/HashtagService.php line 117
    OCA\Social\Service\HashtagService->getTrendSince(1608094202)
11. /var/www/nextcloud/apps/social/lib/Cron/Cache.php line 113
    OCA\Social\Service\HashtagService->manageHashtags()
12. /var/www/nextcloud/apps/social/lib/Cron/Cache.php line 87
    OCA\Social\Cron\Cache->manageCache()
13. /var/www/nextcloud/lib/private/BackgroundJob/Job.php line 52
    OCA\Social\Cron\Cache->run(null)
14. /var/www/nextcloud/lib/private/BackgroundJob/TimedJob.php line 59
    OC\BackgroundJob\Job->execute(OC\BackgroundJob\JobList {}, OC\Log {})
15. /var/www/nextcloud/cron.php line 127
    OC\BackgroundJob\TimedJob->execute(OC\BackgroundJob\JobList {}, OC\Log {})

**To Reproduce**
1. install social (or reset social)
2. post a #1 or new publication
3. error php Array to string conversion

**Client details:**
 - OS: debian
 - Browser: firefox

<details>
<summary>Server details</summary>

**Social app version:**
Social 0.4.2
**Operating system**:
Linux 4.19.0-13-amd64 #1 SMP Debian 4.19.160-2 (2020-11-28) x86_64
**Web server:**
nginx/1.14.2 (fpm-fcgi)
**Database:**
mysql 10.3.27
**PHP version:**
7.3.19-1~deb10u1
Modules loaded: Core, date, libxml, openssl, pcre, zlib, filter, hash, Reflection, SPL, session, sodium, standard, cgi-fcgi, mysqlnd, PDO, xml, apcu, bcmath, calendar, ctype, curl, dom, mbstring, fileinfo, ftp, gd, gettext, gmp, iconv, imagick, imap, intl, json, exif, mysqli, pdo_mysql, apc, posix, readline, shmop, SimpleXML, sockets, sysvmsg, sysvsem, sysvshm, tokenizer, wddx, xmlreader, xmlwriter, xsl, zip, Phar, Zend OPcache
**Nextcloud version:** (see Nextcloud admin page)
20.0.4 - 20.0.4.0
</details>

<details>
<summary>Logs</summary>

#### Nextcloud log (data/nextcloud.log)
```
[PHP] Error: Error: Array to string conversion at /var/www/nextcloud/apps/social/lib/Model/ActivityPub/ACore.php#569 at <<closure>>

 0. <<closure>>
    OC\Log\ErrorHandler::onError(8, "Array to string conversion", "/var/www/nextcl ... p", 569, {as: 8,value: "< ... e})
 1. /var/www/nextcloud/apps/social/lib/Model/ActivityPub/ACore.php line 569
    strip_tags("<p><span class= ... >", ["a","p","span","br"])
 2. /var/www/nextcloud/apps/social/lib/Model/ActivityPub/ACore.php line 498
    OCA\Social\Model\ActivityPub\ACore->validateEntryString(8, "<p><span class= ... >")
 3. /var/www/nextcloud/apps/social/lib/Model/ActivityPub/Stream.php line 428
    OCA\Social\Model\ActivityPub\ACore->validate(8, "content", {id: "https://so ... "}, "")
 4. /var/www/nextcloud/apps/social/lib/Model/ActivityPub/Object/Note.php line 185
    OCA\Social\Model\ActivityPub\Stream->importFromDatabase({id: "https://so ... "})
 5. /var/www/nextcloud/apps/social/lib/Db/StreamRequestBuilder.php line 211
    OCA\Social\Model\ActivityPub\Object\Note->importFromDatabase({id: "https://so ... "})
 6. /var/www/nextcloud/apps/social/vendor/daita/my-small-php-tools/lib/Db/ExtendedQueryBuilder.php line 538
    OCA\Social\Db\StreamRequestBuilder->parseStreamSelectSql({id: "https://so ... "}, OCA\Social\Db\SocialQueryBuilder {})
 7. /var/www/nextcloud/apps/social/lib/Db/StreamRequestBuilder.php line 192
    daita\MySmallPhpTools\Db\ExtendedQueryBuilder->getRows([OCA\Social\Db\S ... "])
 8. /var/www/nextcloud/apps/social/lib/Db/StreamRequest.php line 665
    OCA\Social\Db\StreamRequestBuilder->getStreamsFromRequest(OCA\Social\Db\SocialQueryBuilder {})
 9. /var/www/nextcloud/apps/social/lib/Service/HashtagService.php line 173
    OCA\Social\Db\StreamRequest->getNoteSince(1608094202)
10. /var/www/nextcloud/apps/social/lib/Service/HashtagService.php line 117
    OCA\Social\Service\HashtagService->getTrendSince(1608094202)
11. /var/www/nextcloud/apps/social/lib/Cron/Cache.php line 113
    OCA\Social\Service\HashtagService->manageHashtags()
12. /var/www/nextcloud/apps/social/lib/Cron/Cache.php line 87
    OCA\Social\Cron\Cache->manageCache()
13. /var/www/nextcloud/lib/private/BackgroundJob/Job.php line 52
    OCA\Social\Cron\Cache->run(null)
14. /var/www/nextcloud/lib/private/BackgroundJob/TimedJob.php line 59
    OC\BackgroundJob\Job->execute(OC\BackgroundJob\JobList {}, OC\Log {})
15. /var/www/nextcloud/cron.php line 127
    OC\BackgroundJob\TimedJob->execute(OC\BackgroundJob\JobList {}, OC\Log {})

{"reqId":"k1llsF2U8Z29IhehWeQ5","level":3,"time":"2020-12-26T05:05:02+00:00","remoteAddr":"","user":"--","app":"PHP","method":"","url":"--","message":{"Exception":"Error","Message":"Array to string conversion at /var/www/nextcloud/apps/social/lib/Model/ActivityPub/ACore.php#569","Code":0,"Trace":[{"function":"onError","class":"OC\\Log\\ErrorHandler","type":"::","args":[8,"Array to string conversion","/var/www/nextcloud/apps/social/lib/Model/ActivityPub/ACore.php",569,{"as":8,"value":"<p><span class=\"h-card\"><a href=\"https://mastodon.xyz/@nextcloud\" class=\"u-url mention\">@<span>nextcloud</span></a></span> </p><p>Merry Christmas!</p>","exception":true}]},{"file":"/var/www/nextcloud/apps/social/lib/Model/ActivityPub/ACore.php","line":569,"function":"strip_tags","args":["<p><span class=\"h-card\"><a href=\"https://mastodon.xyz/@nextcloud\" class=\"u-url mention\">@<span>nextcloud</span></a></span> </p><p>Merry Christmas!</p>",["a","p","span","br"]]},{"file":"/var/www/nextcloud/apps/social/lib/Model/ActivityPub/ACore.php","line":498,"function":"validateEntryString","class":"OCA\\Social\\Model\\ActivityPub\\ACore","type":"->","args":[8,"<p><span class=\"h-card\"><a href=\"https://mastodon.xyz/@nextcloud\" class=\"u-url mention\">@<span>nextcloud</span></a></span> </p><p>Merry Christmas!</p>"]},{"file":"/var/www/nextcloud/apps/social/lib/Model/ActivityPub/Stream.php","line":428,"function":"validate","class":"OCA\\Social\\Model\\ActivityPub\\ACore","type":"->","args":[8,"content",{"id":"https://social.linux.pizza/users/StampedingLonghorn/statuses/105441479155641812","nid":"22","type":"Note","subtype":"","to":"","to_array":"[\"https://social.linux.pizza/users/StampedingLonghorn/followers\"]","cc":"[\"https://www.w3.org/ns/activitystreams#Public\",\"https://mastodon.xyz/users/nextcloud\"]","bcc":"[]","content":"<p><span class=\"h-card\"><a href=\"https://mastodon.xyz/@nextcloud\" class=\"u-url mention\">@<span>nextcloud</span></a></span> </p><p>Merry Christmas!</p>","summary":"","attachments":"[]","published":"2020-12-25T15:14:48Z","published_time":"2020-12-25 15:14:48","cache":"[]","object_id":"","attributed_to":"https://social.linux.pizza/users/StampedingLonghorn","in_reply_to":"https://mastodon.xyz/users/nextcloud/statuses/105434454792206340","source":"{\"id\":\"https://social.linux.pizza/users/StampedingLonghorn/statuses/105441479155641812\",\"type\":\"Note\",\"summary\":null,\"inReplyTo\":\"https://mastodon.xyz/users/nextcloud/statuses/105434454792206340\",\"published\":\"2020-12-25T15:14:48Z\",\"url\":\"https://social.linux.pizza/@StampedingLonghorn/105441479155641812\",\"attributedTo\":\"https://social.linux.pizza/users/StampedingLonghorn\",\"to\":[\"https://social.linux.pizza/users/StampedingLonghorn/followers\"],\"cc\":[\"https://www.w3.org/ns/activitystreams#Public\",\"https://mastodon.xyz/users/nextcloud\"],\"sensitive\":false,\"atomUri\":\"https://social.linux.pizza/users/StampedingLonghorn/statuses/105441479155641812\",\"inReplyToAtomUri\":\"https://mastodon.xyz/users/nextcloud/statuses/105434454792206340\",\"conversation\":\"tag:mastodon.xyz,2020-12-24:objectId=72053975:objectType=Conversation\",\"content\":\"<p><span class=\\\"h-card\\\"><a href=\\\"https://mastodon.xyz/@nextcloud\\\" class=\\\"u-url mention\\\">@<span>nextcloud</span></a></span> </p><p>Merry Christmas!</p>\",\"contentMap\":{\"en\":\"<p><span class=\\\"h-card\\\"><a href=\\\"https://mastodon.xyz/@nextcloud\\\" class=\\\"u-url mention\\\">@<span>nextcloud</span></a></span> </p><p>Merry Christmas!</p>\"},\"attachment\":[],\"tag\":[{\"type\":\"Mention\",\"href\":\"https://mastodon.xyz/users/nextcloud\",\"name\":\"@nextcloud@mastodon.xyz\"}],\"replies\":{\"id\":\"https://social.linux.pizza/users/StampedingLonghorn/statuses/105441479155641812/replies\",\"type\":\"Collection\",\"first\":{\"type\":\"CollectionPage\",\"next\":\"https://social.linux.pizza/users/StampedingLonghorn/statuses/105441479155641812/replies?only_other_accounts=true&page=true\",\"partOf\":\"https://social.linux.pizza/users/StampedingLonghorn/statuses/105441479155641812/replies\",\"items\":[]}}}","local":"0","instances":"[]","creation":"2020-12-25 15:14:59","filter_duplicate":"0","details":"[]","hashtags":"[]"},""]},{"file":"/var/www/nextcloud/apps/social/lib/Model/ActivityPub/Object/Note.php","line":185,"function":"importFromDatabase","class":"OCA\\Social\\Model\\ActivityPub\\Stream","type":"->","args":[{"id":"https://social.linux.pizza/users/StampedingLonghorn/statuses/105441479155641812","nid":"22","type":"Note","subtype":"","to":"","to_array":"[\"https://social.linux.pizza/users/StampedingLonghorn/followers\"]","cc":"[\"https://www.w3.org/ns/activitystreams#Public\",\"https://mastodon.xyz/users/nextcloud\"]","bcc":"[]","content":"<p><span class=\"h-card\"><a href=\"https://mastodon.xyz/@nextcloud\" class=\"u-url mention\">@<span>nextcloud</span></a></span> </p><p>Merry Christmas!</p>","summary":"","attachments":"[]","published":"2020-12-25T15:14:48Z","published_time":"2020-12-25 15:14:48","cache":"[]","object_id":"","attributed_to":"https://social.linux.pizza/users/StampedingLonghorn","in_reply_to":"https://mastodon.xyz/users/nextcloud/statuses/105434454792206340","source":"{\"id\":\"https://social.linux.pizza/users/StampedingLonghorn/statuses/105441479155641812\",\"type\":\"Note\",\"summary\":null,\"inReplyTo\":\"https://mastodon.xyz/users/nextcloud/statuses/105434454792206340\",\"published\":\"2020-12-25T15:14:48Z\",\"url\":\"https://social.linux.pizza/@StampedingLonghorn/105441479155641812\",\"attributedTo\":\"https://social.linux.pizza/users/StampedingLonghorn\",\"to\":[\"https://social.linux.pizza/users/StampedingLonghorn/followers\"],\"cc\":[\"https://www.w3.org/ns/activitystreams#Public\",\"https://mastodon.xyz/users/nextcloud\"],\"sensitive\":false,\"atomUri\":\"https://social.linux.pizza/users/StampedingLonghorn/statuses/105441479155641812\",\"inReplyToAtomUri\":\"https://mastodon.xyz/users/nextcloud/statuses/105434454792206340\",\"conversation\":\"tag:mastodon.xyz,2020-12-24:objectId=72053975:objectType=Conversation\",\"content\":\"<p><span class=\\\"h-card\\\"><a href=\\\"https://mastodon.xyz/@nextcloud\\\" class=\\\"u-url mention\\\">@<span>nextcloud</span></a></span> </p><p>Merry Christmas!</p>\",\"contentMap\":{\"en\":\"<p><span class=\\\"h-card\\\"><a href=\\\"https://mastodon.xyz/@nextcloud\\\" class=\\\"u-url mention\\\">@<span>nextcloud</span></a></span> </p><p>Merry Christmas!</p>\"},\"attachment\":[],\"tag\":[{\"type\":\"Mention\",\"href\":\"https://mastodon.xyz/users/nextcloud\",\"name\":\"@nextcloud@mastodon.xyz\"}],\"replies\":{\"id\":\"https://social.linux.pizza/users/StampedingLonghorn/statuses/105441479155641812/replies\",\"type\":\"Collection\",\"first\":{\"type\":\"CollectionPage\",\"next\":\"https://social.linux.pizza/users/StampedingLonghorn/statuses/105441479155641812/replies?only_other_accounts=true&page=true\",\"partOf\":\"https://social.linux.pizza/users/StampedingLonghorn/statuses/105441479155641812/replies\",\"items\":[]}}}","local":"0","instances":"[]","creation":"2020-12-25 15:14:59","filter_duplicate":"0","details":"[]","hashtags":"[]"}]},{"file":"/var/www/nextcloud/apps/social/lib/Db/StreamRequestBuilder.php","line":211,"function":"importFromDatabase","class":"OCA\\Social\\Model\\ActivityPub\\Object\\Note","type":"->","args":[{"id":"https://social.linux.pizza/users/StampedingLonghorn/statuses/105441479155641812","nid":"22","type":"Note","subtype":"","to":"","to_array":"[\"https://social.linux.pizza/users/StampedingLonghorn/followers\"]","cc":"[\"https://www.w3.org/ns/activitystreams#Public\",\"https://mastodon.xyz/users/nextcloud\"]","bcc":"[]","content":"<p><span class=\"h-card\"><a href=\"https://mastodon.xyz/@nextcloud\" class=\"u-url mention\">@<span>nextcloud</span></a></span> </p><p>Merry Christmas!</p>","summary":"","attachments":"[]","published":"2020-12-25T15:14:48Z","published_time":"2020-12-25 15:14:48","cache":"[]","object_id":"","attributed_to":"https://social.linux.pizza/users/StampedingLonghorn","in_reply_to":"https://mastodon.xyz/users/nextcloud/statuses/105434454792206340","source":"{\"id\":\"https://social.linux.pizza/users/StampedingLonghorn/statuses/105441479155641812\",\"type\":\"Note\",\"summary\":null,\"inReplyTo\":\"https://mastodon.xyz/users/nextcloud/statuses/105434454792206340\",\"published\":\"2020-12-25T15:14:48Z\",\"url\":\"https://social.linux.pizza/@StampedingLonghorn/105441479155641812\",\"attributedTo\":\"https://social.linux.pizza/users/StampedingLonghorn\",\"to\":[\"https://social.linux.pizza/users/StampedingLonghorn/followers\"],\"cc\":[\"https://www.w3.org/ns/activitystreams#Public\",\"https://mastodon.xyz/users/nextcloud\"],\"sensitive\":false,\"atomUri\":\"https://social.linux.pizza/users/StampedingLonghorn/statuses/105441479155641812\",\"inReplyToAtomUri\":\"https://mastodon.xyz/users/nextcloud/statuses/105434454792206340\",\"conversation\":\"tag:mastodon.xyz,2020-12-24:objectId=72053975:objectType=Conversation\",\"content\":\"<p><span class=\\\"h-card\\\"><a href=\\\"https://mastodon.xyz/@nextcloud\\\" class=\\\"u-url mention\\\">@<span>nextcloud</span></a></span> </p><p>Merry Christmas!</p>\",\"contentMap\":{\"en\":\"<p><span class=\\\"h-card\\\"><a href=\\\"https://mastodon.xyz/@nextcloud\\\" class=\\\"u-url mention\\\">@<span>nextcloud</span></a></span> </p><p>Merry Christmas!</p>\"},\"attachment\":[],\"tag\":[{\"type\":\"Mention\",\"href\":\"https://mastodon.xyz/users/nextcloud\",\"name\":\"@nextcloud@mastodon.xyz\"}],\"replies\":{\"id\":\"https://social.linux.pizza/users/StampedingLonghorn/statuses/105441479155641812/replies\",\"type\":\"Collection\",\"first\":{\"type\":\"CollectionPage\",\"next\":\"https://social.linux.pizza/users/StampedingLonghorn/statuses/105441479155641812/replies?only_other_accounts=true&page=true\",\"partOf\":\"https://social.linux.pizza/users/StampedingLonghorn/statuses/105441479155641812/replies\",\"items\":[]}}}","local":"0","instances":"[]","creation":"2020-12-25 15:14:59","filter_duplicate":"0","details":"[]","hashtags":"[]"}]},{"file":"/var/www/nextcloud/apps/social/vendor/daita/my-small-php-tools/lib/Db/ExtendedQueryBuilder.php","line":538,"function":"parseStreamSelectSql","class":"OCA\\Social\\Db\\StreamRequestBuilder","type":"->","args":[{"id":"https://social.linux.pizza/users/StampedingLonghorn/statuses/105441479155641812","nid":"22","type":"Note","subtype":"","to":"","to_array":"[\"https://social.linux.pizza/users/StampedingLonghorn/followers\"]","cc":"[\"https://www.w3.org/ns/activitystreams#Public\",\"https://mastodon.xyz/users/nextcloud\"]","bcc":"[]","content":"<p><span class=\"h-card\"><a href=\"https://mastodon.xyz/@nextcloud\" class=\"u-url mention\">@<span>nextcloud</span></a></span> </p><p>Merry Christmas!</p>","summary":"","attachments":"[]","published":"2020-12-25T15:14:48Z","published_time":"2020-12-25 15:14:48","cache":"[]","object_id":"","attributed_to":"https://social.linux.pizza/users/StampedingLonghorn","in_reply_to":"https://mastodon.xyz/users/nextcloud/statuses/105434454792206340","source":"{\"id\":\"https://social.linux.pizza/users/StampedingLonghorn/statuses/105441479155641812\",\"type\":\"Note\",\"summary\":null,\"inReplyTo\":\"https://mastodon.xyz/users/nextcloud/statuses/105434454792206340\",\"published\":\"2020-12-25T15:14:48Z\",\"url\":\"https://social.linux.pizza/@StampedingLonghorn/105441479155641812\",\"attributedTo\":\"https://social.linux.pizza/users/StampedingLonghorn\",\"to\":[\"https://social.linux.pizza/users/StampedingLonghorn/followers\"],\"cc\":[\"https://www.w3.org/ns/activitystreams#Public\",\"https://mastodon.xyz/users/nextcloud\"],\"sensitive\":false,\"atomUri\":\"https://social.linux.pizza/users/StampedingLonghorn/statuses/105441479155641812\",\"inReplyToAtomUri\":\"https://mastodon.xyz/users/nextcloud/statuses/105434454792206340\",\"conversation\":\"tag:mastodon.xyz,2020-12-24:objectId=72053975:objectType=Conversation\",\"content\":\"<p><span class=\\\"h-card\\\"><a href=\\\"https://mastodon.xyz/@nextcloud\\\" class=\\\"u-url mention\\\">@<span>nextcloud</span></a></span> </p><p>Merry Christmas!</p>\",\"contentMap\":{\"en\":\"<p><span class=\\\"h-card\\\"><a href=\\\"https://mastodon.xyz/@nextcloud\\\" class=\\\"u-url mention\\\">@<span>nextcloud</span></a></span> </p><p>Merry Christmas!</p>\"},\"attachment\":[],\"tag\":[{\"type\":\"Mention\",\"href\":\"https://mastodon.xyz/users/nextcloud\",\"name\":\"@nextcloud@mastodon.xyz\"}],\"replies\":{\"id\":\"https://social.linux.pizza/users/StampedingLonghorn/statuses/105441479155641812/replies\",\"type\":\"Collection\",\"first\":{\"type\":\"CollectionPage\",\"next\":\"https://social.linux.pizza/users/StampedingLonghorn/statuses/105441479155641812/replies?only_other_accounts=true&page=true\",\"partOf\":\"https://social.linux.pizza/users/StampedingLonghorn/statuses/105441479155641812/replies\",\"items\":[]}}}","local":"0","instances":"[]","creation":"2020-12-25 15:14:59","filter_duplicate":"0","details":"[]","hashtags":"[]"},{"__class__":"OCA\\Social\\Db\\SocialQueryBuilder"}]},{"file":"/var/www/nextcloud/apps/social/lib/Db/StreamRequestBuilder.php","line":192,"function":"getRows","class":"daita\\MySmallPhpTools\\Db\\ExtendedQueryBuilder","type":"->","args":[[{"__class__":"OCA\\Social\\Db\\StreamRequest"},"parseStreamSelectSql"]]},{"file":"/var/www/nextcloud/apps/social/lib/Db/StreamRequest.php","line":665,"function":"getStreamsFromRequest","class":"OCA\\Social\\Db\\StreamRequestBuilder","type":"->","args":[{"__class__":"OCA\\Social\\Db\\SocialQueryBuilder"}]},{"file":"/var/www/nextcloud/apps/social/lib/Service/HashtagService.php","line":173,"function":"getNoteSince","class":"OCA\\Social\\Db\\StreamRequest","type":"->","args":[1608095102]},{"file":"/var/www/nextcloud/apps/social/lib/Service/HashtagService.php","line":117,"function":"getTrendSince","class":"OCA\\Social\\Service\\HashtagService","type":"->","args":[1608095102]},{"file":"/var/www/nextcloud/apps/social/lib/Cron/Cache.php","line":113,"function":"manageHashtags","class":"OCA\\Social\\Service\\HashtagService","type":"->","args":[]},{"file":"/var/www/nextcloud/apps/social/lib/Cron/Cache.php","line":87,"function":"manageCache","class":"OCA\\Social\\Cron\\Cache","type":"->","args":[]},{"file":"/var/www/nextcloud/lib/private/BackgroundJob/Job.php","line":52,"function":"run","class":"OCA\\Social\\Cron\\Cache","type":"->","args":[null]},{"file":"/var/www/nextcloud/lib/private/BackgroundJob/TimedJob.php","line":59,"function":"execute","class":"OC\\BackgroundJob\\Job","type":"->","args":[{"__class__":"OC\\BackgroundJob\\JobList"},{"__class__":"OC\\Log"}]},{"file":"/var/www/nextcloud/cron.php","line":127,"function":"execute","class":"OC\\BackgroundJob\\TimedJob","type":"->","args":[{"__class__":"OC\\BackgroundJob\\JobList"},{"__class__":"OC\\Log"}]}],"File":"/var/www/nextcloud/lib/private/Log/ErrorHandler.php","Line":91,"CustomMessage":"--"},"userAgent":"--","version":"20.0.4.0","id":"5fe6c48b7fee2"}
```

</details>

**Comment by lstarnes1024** [comment]
I think this is an issue with PHP before 7.4. strip_tags's 2nd parameter, allowed_tags, can be an array or a string or null in 7.4, but 7.3 and earlier only allow allowed_tags to be a string or null.

**Comment by Megachip** [comment]
So fix should be to switch to php 7.4? Same issue on 7.3.28
